### ISSUE TITLE
Improve test robustness

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/PackageManager#47]
GAPDoc updated from 1.6.2 to 1.6.3, so we have a new test failure:
```
########> Diff in /home/gap/inst/gap-stable-4.10/pkg/PackageManager-0.4/tst/Pa\
ckageManager.tst:664
# Input is:
InstallPackage("https://gap-packages.github.io/PackageManager/dummy/uuid-too-n\
ew.tar.gz");
# Expected output:
#I  Package GAPDoc >= 999.0 unavailable: only version 1.6.2 was found
#I  Dependencies not satisfied for uuid-0.6
false
# But found:
#I  Package GAPDoc >= 999.0 unavailable: only version 1.6.3 was found
#I  Dependencies not satisfied for uuid-0.6
false
########
```
(see https://travis-ci.org/gap-infra/gap-docker-pkg-tests-stable-4.10/jobs/559464647). Perhaps one could invent a way to reduce reliance of this test on a specific version number.

**Comment by mtorpey** [comment]
Temporary fix in dc7fcb7, by updating the number.  Permanent fix incoming...

**Comment by mtorpey** [comment]
Permanent fix in cdc353d: we use a hacked package list so that the current GAPDoc version is perpetually 0.2

I'll wait for [Travis tests](https://travis-ci.org/gap-packages/PackageManager/builds/560377340) to pass.

**Comment by mtorpey** [comment]
This passed, and I made a v0.5.1 release so it's available immediately.
